### PR TITLE
Add a method to fetch README.md

### PR DESCRIPTION
**Pull request by akring**
Hi nerdishbynature,

I add a method to fetch README.md contents, this is what I need in my project, and I think this would be helpful for others.

### CODE REVIEW

File: OctoKit/Repositories.swift [hunk 1]
```diff
@@ -44,15 +46,15 @@ import RequestKit
 // MARK: request
 
 public extension Octokit {
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove white space

File: OctoKit/Repositories.swift [hunk 2]
```diff
@@ -44,15 +46,15 @@ import RequestKit
 // MARK: request
 
 public extension Octokit {
-
+    
     /**
-        Fetches the Repositories for a user or organization
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please add the white space again

File: OctoKit/Repositories.swift [hunk 3]
```diff
@@ -44,15 +46,15 @@ import RequestKit
...
-        - parameter completion: Callback for the outcome of the fetch.
-    */
+     Fetches the Repositories for a user or organization
+     - parameter session: RequestKitURLSession, defaults to NSURLSession.sharedSession()
+     - parameter owner: The user or organization that owns the repositories. If `nil`, fetches repositories for the authenticated user.
+     - parameter page: Current page for repository pagination. `1` by default.
+     - parameter perPage: Number of repositories per page. `100` by default.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
     public func repositories(_ session: RequestKitURLSession = URLSession.shared, owner: String? = nil, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Repository]>) -> Void) -> URLSessionDataTaskProtocol? {
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 4]
```diff
@@ -61,21 +63,21 @@ public extension Octokit {
             if let error = error {
                 completion(Response.failure(error))
             }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 5]
```diff
@@ -61,21 +63,21 @@ public extension Octokit {
...
-
+            
             if let json = json {
                 let repos = json.map { Repository($0) }
                 completion(Response.success(repos))
             }
         }
     }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 6]
```diff
@@ -61,21 +63,21 @@ public extension Octokit {
...
         }
     }
-
+    
     /**
-        Fetches a repository for a user or organization
-        - parameter session: RequestKitURLSession, defaults to NSURLSession.sharedSession()
-        - parameter owner: The user or organization that owns the repositories.
-        - parameter name: The name of the repository to fetch.
-        - parameter completion: Callback for the outcome of the fetch.
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please add the white space again

File: OctoKit/Repositories.swift [hunk 7]
```diff
@@ -61,21 +63,21 @@ public extension Octokit {
...
-    */
+     Fetches a repository for a user or organization
+     - parameter session: RequestKitURLSession, defaults to NSURLSession.sharedSession()
+     - parameter owner: The user or organization that owns the repositories.
+     - parameter name: The name of the repository to fetch.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 8]
```diff
@@ -97,23 +126,25 @@ enum RepositoryRouter: Router {
...
     var configuration: Configuration {
         switch self {
         case .readRepositories(let config, _, _, _): return config
         case .readAuthenticatedRepositories(let config, _, _): return config
         case .readRepository(let config, _, _): return config
+        case .readRepositoryReadme(let config, _, _): return config
         }
     }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 9]
```diff
@@ -97,23 +126,25 @@ enum RepositoryRouter: Router {
...
+        case .readRepositoryReadme(let config, _, _): return config
         }
     }
-
+    
     var method: HTTPMethod {
         return .GET
     }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 10]
```diff
@@ -97,23 +126,25 @@ enum RepositoryRouter: Router {
...
     var method: HTTPMethod {
         return .GET
     }
-
+    
     var encoding: HTTPEncoding {
         return .url
     }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 11]
```diff
@@ -122,9 +153,11 @@ enum RepositoryRouter: Router {
             return ["per_page": perPage, "page": page]
         case .readRepository:
             return [:]
+        case .readRepositoryReadme(_, _, _):
+            return [:]
         }
     }
-
+
```
Comment (pietbrauer) [inline OctoKit/Repositories.swift]: Please remove the white space

File: OctoKit/Repositories.swift [hunk 12]
```diff
@@ -89,6 +91,33 @@ public extension Octokit {
...
+                        let decodedString = NSString(data: decodedData, encoding: String.Encoding.utf8.rawValue)!
+                        completion(Response.success(decodedString as String))
+                    }else{
+                        completion(Response.success("" as String))
+                    }
+                }
+            }
+        }
+    }
 }
```
Comment (remirobert) [inline OctoKit/Repositories.swift]: Maybe we should add some unit tests to this one.

**Comment by pietbrauer** [comment]
@remirobert Seems like the origin remote was deleted. Would you be interested in applying the patch to a branch and add the suggested modifications in a new Pull Request?

```bash
$ git checkout -b akring-master master
$ curl -L https://github.com/nerdishbynature/octokit.swift/pull/61.patch | git am -3
```

**Comment by pietbrauer** [comment]
Closing this because of inactivity.

**Pull request closed without merging**